deployAWS/push2lambda.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# note that this script needs to be run from the deployAWS directory to
+# zip up the end-to-end script and its local dependencies.
+# I could have listed a machine specific absolute path for the first
+# cd command but I wanted to give this a chance to work on yours without editing;
+# therefore I went with relative paths assuming I only have root and below to work from.
+
+# temporarily copy Amazon linux compatible libraries to the source code directory
+# before zipping everything up. When I don't do this, the libraries wind up in their own
+# folder in the zip archive which doesn't work well when uploaded to a Lambda. 
+libs="numpy pandas pytz"
+cd ../lib
+cp -r ${libs} ../src
+
+# create the zip file of source code for my Lambda
+cd ../src
+zip -r SampleProject.zip lambda_function.py Transform.py Load.py Extract.py ${libs}
```

Comment:
Once you've created `setup.py` and `requirements.txt`, please create a separate `scripts/bundle.sh` shell script that looks similar to this:

```
#!/usr/bin/env bash

set -e

ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/.."
echo "ROOT_DIR=${ROOT_DIR}"
BUILD_DIR=$(mktemp -d)

pip install -r $ROOT_DIR/requirements.txt --target $BUILD_DIR
cp -R ${ROOT_DIR}/myetl $BUILD_DIR

mkdir -p $ROOT_DIR/dist
rm -f $ROOT_DIR/dist/myetl-lambda-function.zip

# Clean Python intermediates.
find $BUILD_DIR -name '*.pyc' -delete
find $BUILD_DIR -name '__pycache__' -delete
find $BUILD_DIR -name '.pytest_cache' -delete

cd $BUILD_DIR && zip -r9 $ROOT_DIR/dist/myetl-lambda-function.zip .
rm -rf $BUILD_DIR

echo "Created file: dist/myetl-lambda-function.zip"
```